doc/flame/game.md

```diff
@@ -1,3 +1,46 @@
+# GameWidget
+
+A `GameWidget` is the Flutter `Widget` that is used to insert a `Game` inside the flutter widget
+tree.
+
+It can directly receive a `Game` instance by using its default constructor or it can receive a
+`GameCreate` function on the `controlled` constructor that will be used to create the game
+once the `GameWidget` is inserted in the widget tree.
+
+Example:
```

Comment:
```suggestion
The `GameWidget` is a Flutter `Widget` that is used to insert a `Game` inside the Flutter widget
tree.

It can directly receive a `Game` instance in its default constructor or it can receive a `GameCreate`
function on the `controlled` constructor that will be used to create the game once the `GameWidget`
 is inserted in the widget tree.

Example:
```

Reply:
~~`GameCreate`~~ `create`

Reply:
Changed (I suggested to change that overall though, so might change again)